### PR TITLE
Changes from Request

### PR DESCRIPTION
**Pull request by Que-tin**
[fix] Total props count test
[chg] Remove duplicate media definitons

### CODE REVIEW

File: src/props.animations.js
Comment (saitheninja) [inline src/props.animations.js:1]: looks like this was just a linter thing?

**Comment by saitheninja** [comment]
yo @Que-tin. thanks for the pull.

i ran the build and the generated artifacts look correct, and all under one media query. also the tests passed.

everything looks good to me except for a little linting. please could you clean that up? then i'll go ahead and merge 👍🏾

edit: whoops wrong button. didn't mean to close the pr.

File: src/props.animations.js
Comment (Que-tin) [reply src/props.animations.js:1]: Mhm? What do you mean? I don't see what you are taling about

Comment (saitheninja) [reply src/props.animations.js:1]: some small spacing changes in src/props.animations.js. but we didn't change the fade-out-bloom animation, so i don't think anything needs to get changed?

sorry, it's a nitpick, i know. i just want to keep the git history as clean as i can.

Comment (Que-tin) [reply src/props.animations.js:1]: Oh yeah got it. Done. Can be merged now. :)

Comment (saitheninja) [reply src/props.animations.js:1]: hey @Que-tin, thanks for making the changes. also sorry for leaving you hanging for so long. i will go ahead and merge this now!

**Review by saitheninja — APPROVED** [review]

**Pull request merged**